projects/angular-custom-pipes/src/lib/array/is-empty/is-empty.pipe.ts

```diff
@@ -0,0 +1,10 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'isEmpty'
+})
+export class IsEmptyPipe implements PipeTransform {
+  transform(value: any[]): any {
```

Comment:
czy ten pipe nie pownien zawracać booleana?

Reply:
jesli user bedzie wykorzystywał pipe'a na tablicy to jak najbardziej, pytanie co jeśli przekaże np obiekt, w tym przypadku zwracam obiekt,  co rekomendujesz w takim przypadku

Reply:
a czy ideą tego pipe nie jest informacja, czy to co wrzucisz jest puste czy nie?
Spojrzałem na https://lodash.com/docs/#isEmpty
![image](https://user-images.githubusercontent.com/25820809/77396558-44874000-6da4-11ea-8130-6c2afd30382f.png)